examples/python/plot_projdata_profiles.py

```diff
@@ -0,0 +1,305 @@
+        if isinstance(f, str):
+            label = f
+        else:
+            label = ""
+
+        projdata_npy = get_projection_data_as_array(f)
+        if projdata_npy is None:
+            continue
+
+        # Generate the 1D array
+        try:
+            plot_data = compress_and_extract_1d_from_nd_array(projdata_npy, display_axis, data_indices)
+        except ValueError as e:
+            print(f"Error generating 1D array object.\nError message: {e}")
+            continue
+
+        plt.plot(plot_data, label=label)
+
+    if len(plt.gca().get_lines()) == 0:
+        print("Something went wrong! No data to plot.")
+        return
+
+    # Identify sum and extraction axes
+    sum_axis = [i for i, x in enumerate(data_indices) if x is None and i != display_axis]
+    index_axis = [i for i, x in enumerate(data_indices) if x is not None and i != display_axis]
+
+    # Extract labels and values for sum and extraction axes
+    sum_axis_labels = [PROJDATA_DIM_MAP[i] for i in sum_axis]
+    extraction_axis_labels = [PROJDATA_DIM_MAP[i] for i in index_axis]
+    index_values = [data_indices[i] for i in index_axis]
+
+    # Plot title
+    plt.title(f"Summing {sum_axis_labels} axis and extracting {extraction_axis_labels} with values {index_values}")
+    plt.xlabel(f"{PROJDATA_DIM_MAP[display_axis]}")
+    ax.legend()
+    plt.show()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(sys.argv[0])
+    parser.description = ("This script loads, sums axis' and plots profiles over input projection data files."
+                          "The default is to sum over all components, except the display axis."
+                          "The indices used are array based, not STIR offset based.")
+    parser.add_argument('filenames',
+                        nargs='*',
+                        help='Projection data file names to show, can handle multiple.')
+    parser.add_argument('--display_axis',
+                        dest="display_axis",
+                        type=int,
+                        default=3,
+                        help='The horizontal component of the projection data to plot.'
+                             'The default is -1 indicating a sum over all components. '
+                             '0: TOF, 1: axial (and segment), 2: view, 3: tangential.')
+    parser.add_argument('--tof',
+                        dest="tof",
+                        type=int,
+                        default=None,
+                        help='The TOF value of the projection data to plot.'
+                             'The default is to sum over all TOF values.')
+    parser.add_argument('--axial_segment',
+                        dest="axial_segment",
+                        type=int,
+                        default=None,
+                        help='The axial segment number of the projection data to plot.'
+                             'The default is to sum over all axial segments.')
```

Comment:
Axial and segment data are collapsed into the same dimension of the data, this means that it isn't easy to show only axial components.

Reply:
Agreed. However, many users will not be so surprised by this, as the result of `to_numpy` has this "feature" as well. It's also how GE and Siemens store data. So I think we just leave this as-is.